js/stanford_paragraph_types.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * Applies some functionality on the paragraph types.
+ */
+
+(function ($) {
+  Drupal.behaviors.stanfordParagraphTypes = {
+    attach: function (context, settings) {
+      $('.field-name-field-p-hero-image').once(function () {
+        if ($(this).siblings('.field-name-field-p-hero-video').length) {
+          var play = $('<a>', {
+            class: 'play-video',
+            href: '#',
+            onclick: 'return false;',
+            html: $('<i>', {class: 'fa fa-youtube-play icon-youtube-play', html: 'Play'})
+          });
+          $(this).prepend(play);
+          $(play).click(function () {
+            $(this).parent().hide();
```

Comment:
each time you use `$(this).parent()` javascript has to navigate the DOM and can sometimes be slow. You can store a reference to the element you are working with and then call actions on the element as a micro-performance improvement.

eg: 
```
$dad = $(this).parent();
$dad.hide();
$data.siblings(...);
```